openlibrary/macros/LoanStatus.html

```diff
@@ -44,14 +43,15 @@
       <input type="submit" value="Return eBook" class="submit unwait-btn" id="return_ebook"/>
     </form>
 
-$elif (availability == 'borrow_available') or my_turn_to_borrow:
+$elif (availability_status == 'borrow_available') or my_turn_to_borrow:
     <p>
       <a href="$borrow_link" title="Borrow from $contributor" id="borrow_ebook" class="borrow-btn borrow-link">
         Borrow eBook
       </a>
     </p>
 
-$elif (availability == 'borrow_unavailable'):
+$elif (availability_status == 'borrow_unavailable'):
+    $ wlsize = availability.get('num_waitlist', 0)
```

Comment:
QUESTION: Could it not perhaps lead to a misleading user experience, if failing to procure waitlist info is treated as a zero waitlist scenario? ie. "no news" is not "good news" but simply "no news", which the UI should reflect accordingly (i.e. in other places in this PR).